src/lime/_internal/backend/native/NativeAudioSource.hx

```diff
@@ -294,9 +294,10 @@ class NativeAudioSource
 
 			AL.sourceQueueBuffers(handle, numBuffers, buffers);
 
-			// If openAL runs out of buffer it will stop playback.
-			// This check is here to recover from this and resume playback
-			// this situation typically happens when resizing window or other operations that freezes the main thread
+			// OpenAL can unexpectedly stop playback if the buffers fill up,
```

Comment:
Not sure this is the right wording.
There's a loop that make sure there is always data to be read  for the audio engine, when the main thread "freezes" this loop stops feeding  the buffers so  it runs out of data. it stops playing because there's no more data to be read, not because the buffers are filled with data.

Reply:
Oh. That's probably true now that I take another look at it. I was trying to keep the original meaning, but I interpreted "runs out of buffer" to mean "runs out of space in the buffer" (full), but it could also mean "runs out of data in the buffer" (empty).